pinot-common/src/test/java/org/apache/pinot/sql/parsers/CalciteSqlCompilerTest.java

```diff
@@ -422,4 +422,36 @@ public void testSelectionTransformFunction() {
     Assert.assertEquals(
         pinotQuery.getFilterExpression().getFunctionCall().getOperands().get(1).getLiteral().getStringValue(), "v1");
   }
+
+  @Test
+  public void testSqlBaseArith() {
```

Comment:
Could you also add tests for nested expressions, for example: `(a * ((b  - c) / d) + e)`

Reply:
Thanks for review, @fx19880617 , @kishoreg , @mayankshriv , add a nested expression test case.